mreg_cli/commands/host_submodules/core.py

```diff
@@ -141,13 +141,13 @@ def add(args: argparse.Namespace) -> None:
             try:
                 network = Network.get_by_ip(ipaddr)
                 if network:
```

Comment:
```suggestion
                if network and not force:
```

I considered putting the force check here instead, but I think the more explicit approach I took with force checks for each case is more robust, since we may want to add more checks in the future that are not dependent on force status.